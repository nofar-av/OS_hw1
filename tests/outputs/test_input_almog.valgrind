==27320== Memcheck, a memory error detector
==27320== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27320== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27320== Command: /home/omer/OS_wet1/smash
==27320== Parent PID: 27315
==27320== 
==27320== 
==27320== FILE DESCRIPTORS: 4 open at exit.
==27320== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_input_almog.valgrind
==27320==    <inherited from parent>
==27320== 
==27320== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_input_almog.err
==27320==    <inherited from parent>
==27320== 
==27320== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_input_almog.out
==27320==    <inherited from parent>
==27320== 
==27320== Open file descriptor 0:
==27320==    <inherited from parent>
==27320== 
==27320== 
==27320== HEAP SUMMARY:
==27320==     in use at exit: 0 bytes in 0 blocks
==27320==   total heap usage: 35 allocs, 35 frees, 82,667 bytes allocated
==27320== 
==27320== All heap blocks were freed -- no leaks are possible
==27320== 
==27320== For counts of detected and suppressed errors, rerun with: -v
==27320== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
