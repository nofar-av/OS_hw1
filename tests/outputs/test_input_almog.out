smash> smash: got ctrl-Z
smash: process 5600 was stopped
smash> [1] sleep 10 : 5600 0 secs (stopped)
smash> sleep 10 : 5600
smash> [1] sleep 10 : 5600 0 secs
smash> sleep 10 : 5600
smash: got ctrl-C
smash: process 5600 was killed
smash> smash: sending SIGKILL signal to 0 jobs:
